packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/base/data_sources/remote/http_clients/api_http_client.dart

```diff
@@ -29,4 +40,18 @@ class ApiHttpClient with DioMixin implements Dio {
       analyticsInterceptor,{{/analytics}}
     ]);
   }
+  {{#enable_dev_menu}}
+
+  Future<void> setupProxy() async {
+    String proxy = await _storage.getString('proxy') ?? '';
```

Comment:
- I believe if the proxy is not added to shared preferences the http client shouldn't configure its proxy